### ISSUE TITLE
Goldwater Scholarship news item for Karan

### ISSUE THREAD

**Issue by ngehlenborg** [hms-dbmi/gehlenborglab-website#251]

**Comment by ngehlenborg** [comment]
Include link to https://news.umbc.edu/umbc-students-set-new-record-in-prestigious-goldwater-scholarships-for-stem-research/

**Comment by ngehlenborg** [comment]
Yes, would be great if we can add a brief news item about that (but set the date to 2021 ...) and link it to Karan's profile.

**Comment by thomcsmits** [comment]
Closed with https://github.com/hms-dbmi/gehlenborglab-website/pull/765
